src/becca/entities/boption.ts

```diff
@@ -16,10 +16,12 @@ class BOption extends AbstractBeccaEntity<BOption> {
     value!: string;
     isSynced!: boolean;
 
-    constructor(row: OptionRow) {
+    constructor(row?: OptionRow) {
         super();
 
-        this.updateFromRow(row);
+        if (row) {
```

Comment:
Although I generally agree with this approach, this is not the case here since it changes the branching of the constructor.

Currently the next line:

```
this.becca.options[this.name] = this;
```

is executed both when the row is defined and undefined. Whereas if we add a guard condition then this will no longer be called when the row is undefined.